activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -150,6 +151,10 @@ def uuid(*args, **options)
         def xml(*args, **options)
           args.each { |name| column(name, :xml, options) }
         end
+
+        def uuid_primary_key_default
+          ActiveRecord::Base.connection.supports_pgcrypto? ? 'gen_random_uuid()' : 'uuid_generate_v4()'
```

Comment:
@y-yagi `pgcrypto` is available in pg version older than `9.4` (`8.3` for example). But it couldn't be used for UUIDs because `gen_random_uuid` was only introduced in `9.4`